R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -427,24 +1746,78 @@ uby_rolling_avg_tick <- function(cols, rev_ticks, fwd_ticks = 0) {
   return(UpdateByOp$new(INTERNAL_rolling_avg_tick(cols, rev_ticks, fwd_ticks)))
 }
 
+#' @name
+#' uby_rolling_avg_time
+#' @title
+#' Rolling average with time as the windowing unit
+#' @md
+#'
 #' @description
 #' Creates a simple moving average (SMA) UpdateByOp for each column in `cols`, using time as the windowing unit.
-#' This uses ISO-8601 time strings as the reverse and forward window parameters.
+#'
+#' @details
+#' This uses [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) time strings as the reverse and forward window parameters.
```

Comment:
It supports these and other formats.  I think the examples below fall into the other formats.